fluent-package/manage-fluent-repositories.sh

```diff
@@ -44,6 +45,12 @@ case $COMMAND in
 	    exit 1
 	fi
 	;;
+    download-artifacts)
+	# Given URL will not be used. Just use the number of pull request.
+	# Allow copying the URL from browser's URL bar.
+	PULL_REQUEST_URL=$2
+	PULL_NUMBER=${PULL_REQUEST_URL##*/}
```

Comment:
How should we specify `GITHUB_ACCESS_TOKEN`?
That value is very secret, so we should be careful about specifying it.

I think this would be convenient.

```suggestion
	PULL_REQUEST_URL=$2
	PULL_NUMBER=${PULL_REQUEST_URL##*/}
	read -rsp "Please enter your GitHub personal access token: " GITHUB_ACCESS_TOKEN
	echo
```

What do you think?

Reply:
Thanks, it seems reasonable change.